README.rst

```diff
@@ -128,60 +128,31 @@ file or disable package repositories update in config.yaml.
 Using the RPMs
 --------------
 
-We decided that's is a little bit intrusive to install all the
-produced RPMs since some of them may not fit your needs. At the end of
-the process you'll see a new ``result`` directory inside this
-project's root.
+At the end of the process you'll see a new ``result`` directory inside
+this project's root. It contains the built packages repositories under
+``packages`` and yum repository files under ``repository_files`` to
+simplify using those repositories.
 
-A suggested set of packages tested is the following:
+To install only virtualization related packages, for example, you can
```

Comment:
- You are altering 2 things in the same patch: mentioning the new yum repository config file and mentioning the new metapackages. IMHO the latter should be in a separate patch.
- It'd be nice to mention the list of RPM packages installed by each metapackage in some doc (AFAIK it is not) and not force the user to read rpm .spec files to figure it out.